backend/src/data/models/appointment.ts

```diff
@@ -0,0 +1,34 @@
+import { Sequelize, DataTypes, ModelCtor, Model } from 'sequelize';
+
+enum Type {
+  ONLINE = 'online',
+  OFFLINE = 'offline'
+}
+
+interface AppointmentInstance extends Model {
+  date: Date;
+  type: Type;
+  cost: number;
+  subject: string;
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+export default (orm: Sequelize): ModelCtor<AppointmentInstance> => {
```

Comment:
Let's avoid any anonymous functions with the export and let's do any export only at the bottom of the file